forge/routes/api/team.js

```diff
@@ -63,6 +63,32 @@ module.exports = async function (app) {
         }
     })
 
+    app.post('/check-slug', {
```

Comment:
[nb] Not needed now, but I wonder if we should split the routes with a comment for member and collection endpoints.

(check-slug and `/` are collection routes, anything with `/:teamId/` is a member route)

Reply:
Yup - something for another day, but not really worth tracking in an issue.